.ci/jenkins/Jenkinsfile.post-release

```diff
@@ -116,6 +116,31 @@ pipeline {
                 }
             }
         }
+       stage('update kie-benchmarks main to next SNAPSHOT version') {
+            steps {
+                script {
+                    String prLink = null
+                    String kieBenchmarksRepo = 'kie-benchmarks'
+                    String commitMsg = 'bumped up kie-version to next SNAPSHOT'
+                    String prBranch = 'kie-benchmarks-update'
+                    String targetBranch = 'main'
+                    dir("$kieBenchmarksRepo-bot"){
+                        checkoutRepo(kieBenchmarksRepo, targetBranch)
+                        githubscm.setUserConfigFromCreds()
+                        githubscm.createBranch(prBranch)
+                        // update version.org.optaplanner version in kie-benchmarks
+                        vars.mvnSetVersionProperty('version.org.optaplanner',${getNextMinorSnapshotVersion(getProjectVersion())})
```

Comment:
I guess we can use here  maven. reference like it is done here : https://github.com/dupliaka/optaplanner/blob/5e36aecfa6b18b6f36e4e019fed5b70449176fea/.ci/jenkins/Jenkinsfile.promote#L473